rails-snow-queen/app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,69 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS and SCSS file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ */
+$theme-colors: (
+  "primary":#61D5DD,
+  "warning":#fa7f28,
+  "success":#76B041,
+  "info": #99C5E6,
+  "secondary": #FFC914,
+  "light": #F6F7F7,
+  "dark": #177E89
+);
+
+@import 'bootstrap';
+
+
+header {
+  background-color: rgba(97, 213, 221, 0.452);
+  height: 20vh;
+}
+
+#map {
+  width: 100%;
+  height: 80vh;
+  background-color: grey;
+}
+
+#panel {
+  background-color: #177E89;
+}
+
+.map-btn {
```

Comment:
Curious why so many changes for this button? What is special about it?

Reply:
it is just the one custom button I added to the map to remove the polygons, I was trying to imitate the google look, I don't think I'll have more than one, should I still change it?

Reply:
If you change the [node for the control](https://github.com/rosadiaz/snow_queen_v2/pull/1/files/4a57a1d4c8f2c27b1e9d949b2e0df9176b93d071..d30e1feec620879e3b4d73197e090ad900bf0932#diff-18de7bca6b7ed05fe0462069b06296e6R62) from `div` to `button`, you might need to do a bit less styling.